SteamKit2/Tests/DebugLogFacts.cs

```diff
@@ -106,5 +106,20 @@ public void DebugLogCanWriteSafelyWithoutParams()
 
             DebugLog.WriteLine( "category", "msg{0}msg" );
         }
+
+        [Fact, DebugLogSetupTeardown]
+        public void DebugLogFormatsParams()
+        {
+            DebugLog.Enabled = true;
+            DebugLog.AddListener( ( category, msg ) =>
+            {
+                Assert.Equal( "category", category );
+                Assert.Equal( "msg1msg2", msg);
```

Comment:
![](https://user-images.githubusercontent.com/613331/52261349-ccf11a00-2931-11e9-88b6-0815ebbbb4cc.jpg)

Reply:
Maybe I _should_ just commit #382 in the interim...